packages/sdk-react/test/useGraph.test.ts

```diff
@@ -134,3 +135,152 @@ describe("useGraph() hook tests", () => {
     );
   });
 });
+
+describe("useGraphWithCredential() hook tests", () => {
+  beforeEach(() => {});
+
+  afterEach(() => {
+    jest.resetAllMocks();
+    jest.clearAllMocks();
+  });
+
+  it("call function after initialized", async () => {
+    jest
+      .spyOn(teamsfxlib, "TeamsUserCredential")
+      .mockImplementation(
+        (authConfig: teamsfxlib.AuthenticationConfiguration): teamsfxlib.TeamsUserCredential => {
+          return {
+            async login(): Promise<any> {},
+            async getToken(): Promise<any> {},
+            async getUserInfo(): Promise<any> {},
+          };
+        }
+      );
+
+    let authenticatedGraph: Client | undefined;
+    let graphScope: string[] | undefined;
+    console.log("aaa");
```

Comment:
It might be better to remove logs to keep the display results clean

Reply:
Oh, thanks~